test/elixir_search_extractor/file_upload_test.exs

```diff
@@ -6,11 +6,16 @@ defmodule ElixirSearchExtractor.FileUploadTest do
   alias ElixirSearchExtractor.FileUpload
   alias ElixirSearchExtractor.FileUpload.KeywordFile
 
-  describe "list_keyword_files/0" do
-    test "returns all keyword_files" do
-      keyword_file = insert(:keyword_file)
+  describe "paginated_user_keyword_files/2" do
+    test "returns only user's keyword_files" do
+      user = user_fixture()
+      user_file = insert(:keyword_file, user: user)
```

Comment:
Should we add another user keyword file to make sure it is not included in the result?

Reply:
@rosle added in [14aebd7](https://github.com/rafayet-monon/elixir_search_extractor/pull/36/commits/14aebd7020bfc582d397d624e5346dc1ea563b8a)